test/messages/message.js

```diff
@@ -16,7 +16,7 @@ describe('Message', function() {
 
   describe('#toBuffer', function() {
     it('serialize to a buffer', function() {
-      var message = new Message({magicNumber: 0xd9b4bef9, command: 'command'});
+      var message = new Message(null, {magicNumber: 0xd9b4bef9, command: 'command'});
```

Comment:
I don't see the need for this extra `null`, maybe remove the first parameter? (see previous comment)